Source/ActionController.swift

```diff
@@ -312,17 +325,33 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
     
     open func calculateContentInset() {
         // calculate content Inset
-        collectionView.layoutSubviews()
-        if let section = _sections.last, !settings.behavior.useDynamics {
-            let lastSectionIndex = _sections.count - 1
-            let layoutAtts = collectionViewLayout.layoutAttributesForItem(at: IndexPath(item: section.actions.count - 1, section: hasHeader() ? lastSectionIndex + 1 : lastSectionIndex))
-            contentHeight = layoutAtts!.frame.origin.y + layoutAtts!.frame.size.height
+        if collectionViewLayout.dynamicAnimator != nil {
```

Comment:
please consider using `settings.behavior.useDynamics` instead of `collectionViewLayout.dynamicAnimator != nil`